packages/wrangler/src/environment-variables/misc-variables.ts

```diff
@@ -273,15 +273,3 @@ export const getDockerPath = getEnvironmentVariableFactory({
 		return "docker";
 	},
 });
-
-/**
- * `WRANGLER_DOCKER_HOST` specifies the Docker socket to connect to.
- */
-export const getDockerHost = getEnvironmentVariableFactory({
-	variableName: "WRANGLER_DOCKER_HOST",
```

Comment:
Technically I think you can delete `WRANGLER_DOCKER_HOST` from `type VariableNames` after this change.

But I think it is good to have all used variables there, so maybe:

- you should add a comment that while used in in misc-variables, the variable is still used to retrieve the host
- you should also add `DOCKET_HOST` there for the same reason of documenting the env vars used by wrangler.